pos_warning_exiting/static/src/js/chrome.js

```diff
@@ -0,0 +1,87 @@
+odoo.define("pos_warning_exiting.Chrome", function (require) {
+    "use strict";
+
+    const {_t} = require("web.core");
```

Comment:
You can just use `this.env._t` and avoid this requirement

Not blocking though